Functions.Templates/Templates/SignalRConnectionInfoHttpTrigger-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,97 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+		    "Http",
+        "SignalR"
+    ],
+    "name": "SignalRTrigger",
+    "identity": "Azure.Function.CSharp.SignalRTrigger.2.x",
+    "groupIdentity": "Azure.Function.SignalRTrigger",
+    "shortName": "signalr",
+    "tags": {
+        "language": "C#",
+        "type": "item"
+    },
+    "sourceName": "SignalRTriggerCSharp",
```

Comment:
This should be SignalRConnectionInfoHttpTriggerCSharp, same as the .cs file name